pkg/service/init.go

```diff
@@ -323,6 +323,12 @@ func RunE2E(clientset *kubernetes.Clientset) {
 // Cleanup removes all resources created during E2E tests.
 func Cleanup(clientset *kubernetes.Clientset) {
 	namespace := viper.GetString("namespace")
+    
+	if namespace == "" {
```

Comment:
@Bharadwajshivam28 A cleaner solution would be to reuse the ValidateArgs method https://github.com/kubernetes-sigs/hydrophone/blob/main/pkg/common/args.go#L52. its currently validating multiple things. maybe we can introduce a new method for validating namespace?

Reply:
Okay works for me... let me change it @reetasingh